src/lib/event.ts

```diff
@@ -175,10 +176,13 @@ const getEventsByDate = async (startDate: number, endDate: number): Promise<Even
   const keys = byStart.filter(x => byEndSet.has(x));
 
   const events: JsonEvent[] = (await getObjects(keys)).filter(Boolean);
-  const cids = await getCidsByPids(events.map(event => event.pid));
+  const pids = events.map(event => +event.pid);
```

Comment:
Same deal here, move `.map(fixEvent)` ahead and then pid will be a number.

Reply:
will change every force-typing occurences :)

Reply:
The only variable to be typecast is in the `parsePost` method, I think.